Engine/robot.py

```diff
@@ -4,10 +4,10 @@
 from Util.geometry import wrap_to_pi
 
 
-MAX_LINEAR_SPEED = 3000  # mm/s
-MAX_LINEAR_ACCELERATION = 3000  # mm/s^2
-MAX_ANGULAR_SPEED = 5  # rad/s
-MAX_ANGULAR_ACCELERATION = 2  # rad/s^2
+MAX_LINEAR_SPEED = 4000  # mm/s
+MAX_LINEAR_ACCELERATION = 4000  # mm/s^2
+MAX_ANGULAR_COMMAND = 100  # rad
+MAX_ANGULAR_COMMAND_VARIATION = 3  # rad/s
```

Comment:
Not use

Reply:
bah c'est logique d'avoir les propriété là. maybe elle vont se faire utiliser someday,